index.html

```diff
@@ -318,23 +318,23 @@ <h2 class="contacts_heading">Contact me</h2>
               love to help with it
             </p>
           </div>
-          <form action="https://formspree.io/f/{form_id}" method="post">
+          <form action="https://formspree.io/f/mrgjjgzn" method="post">
             <input
               class="form-input"
-              type="text"
+              name="text"
               placeholder="Roberto Firmino"
               maxlength="30"
               required
             />
             <input
               class="form-input"
-              type="email"
+              name="email"
               placeholder="bobbyfirmino@gmail.com"
               required
             />
```

Comment:
- [ ] Great job making the changes @mutugi-c, But while doing so I notice you have removed the `type` attributes from your form elements, and as part of this milestone it's among the main requirement that `all input fields have the correct TYPPE attribute`. This has a lot of advantages when it comes to accessibility.  Please make sure the name has a `text` and the email has an `email` type.